notebooks/03-query/16-export-msq-query.ipynb

```diff
@@ -0,0 +1,426 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0cb3b009-ebde-4d56-9d59-a028d66d8309",
+   "metadata": {},
+   "source": [
+    "# Exporting Data \n",
```

Comment:
Using INSERT with EXTERN to export data (experimental)

(a) A search / SEO thing plus also just makes it clear what functions is being used to do what
(b) = making clear that this is experimental

Reply:
done